snmp/datadog_checks/snmp/data/default_profiles/barracuda-cloudgen.yaml

```diff
@@ -141,13 +141,14 @@ metrics:
           OID: 1.3.6.1.4.1.10704.1.7.1.2
           name: bgpNeighborState
         tag: bgp_neighbor_state
-        mapping:
-          1: idle
-          2: connect
-          3: active
-          4: opensent
-          5: openconfirm
-          6: established
+        # mapping:
+        #   1: idle
+        #   2: connect
+        #   3: active
+        #   4: opensent
+        #   5: openconfirm
+        #   6: established
+        #  mappings not found in MIB
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏If the mapping for `bgpNeighborState` is not known, I think it might be better to completely comment the tag for now.

If we keep the metric without mapping, it will be a breaking change when we add it later or we will need to use a different name.